test/LibraryManager.Test/Providers/Unpkg/UnpkgCatalogTest.cs

```diff
@@ -154,5 +152,16 @@ public async Task GetLatestVersion_PreRelease()
                 Assert.AreNotEqual(existing[1], result);
             }
         }
+
+        [TestMethod]
+        public async Task GetLibraryCompletionSetAsync_WithNullNpmPackageInfoVersions()
```

Comment:
just a nit: I've been using the naming scheme `<MethodBeingTested>_<DescribeInputScenario>_<DescribeExpectedResult>` where the last portion usually begins with a verb like Return or Should.  That way it's clear simply from the test name what the intent of all of the Assert statements are, and you can often tell which scenario is broken without even looking at the test implementation.